schema/src/main/java/org/hl7/v3/RCMRMT030101UK04SpecimenMaterial.java

```diff
@@ -52,7 +52,7 @@
 public class RCMRMT030101UK04SpecimenMaterial {
 
     @XmlElement(required = true)
-    protected ED desc;
+    protected String desc;
```

Comment:
Could you add this class to the list of manually edited ones in the schema README? If its not there already

Reply:
It's there :)